lib/Core/Executor.cpp

```diff
@@ -1415,8 +1429,23 @@ static inline const llvm::fltSemantics * fpWidthToSemantics(unsigned width) {
   }
 }
 
+
 void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
   Instruction *i = ki->inst;
+
+  //
+  if (interpreterOpts.TaintConfig == 3){
```

Comment:
The thing is that those are in an order. NOTAINT << DIRECT << INDIRECT << INDIRECT-SESE
Each class including the previous functionality. Can we still do this with enums? example?